src/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Route, BrowserRouter } from 'react-router-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer, { defaultRootState } from './reducers/index';
+import Navbar from './components/navbar';
+import Footer from './components/footer';
+import LandingPage from './components/landingPage';
+import dashboard from './components/dashboard';
+import uploadScreen from './components/uploadScreen';
+
+require('bootstrap');
```

Comment:
Alternatively, we can use `import 'bootstrap';`

See below for reference:
https://getbootstrap.com/docs/4.0/getting-started/webpack/#importing-javascript